dkg/dkg.go

```diff
@@ -565,3 +569,20 @@ func waitConnect(ctx context.Context, tcpNode host.Host, p peer.ID) (time.Durati
 
 	return 0, false
 }
+
+func forkVersionToNetwork(forkVersion string) (string, error) {
```

Comment:
@corverroos do you think functions like `forkVersionToNetwork()` and `networkToForkVersion()` should be separated out somewhere?

Reply:
This is very small piece of code, can be duplicated imo

Reply:
Indeed, also remember the rule of 3, only once it is used in 3 places or more, is it worth extracting.